### PR TITLE
Issue 2 - Adjust Root to Redirect to '/reminders'

### LINKED ISSUE #2: Adjust Root to Redirect to ‘/reminders’
When the user visits the root of the application, they should be redirected to '/reminders' and should see a list of all reminders on the page. This means making a hopefully small adjustment to your router.js file.

**Hot Tip**: This will make your previous test fail (if you had it passing). Make the necessary changes in the test(s).

### PR DESCRIPTION
**Pull request by dylanavery720**
@martensonbj @Tman22 @adam-rice 

## Purpose

[Link to Issue](https://github.com/turingschool-projects/1610-remember-2/issues/2)

User is redirected to '/reminders' path when visiting the '/' path of the Application.

Closes #2 :boxing_glove: 

## Approach

We used routing and 'transtitionTo', to make a redirect.

### Learning

We read the Ember docs regarding routing and redirecting with 'transtitionTo', and we used example syntax from the Ember docs.

### Test coverage 

Test were provided in boilerplate and tests are now passing after appropriate adjustments were made.

### Merge Dependencies and Related Work

_Link to merge dependency issues or PRs._

### Follow-up tasks

- [Issue #3](https://github.com/turingschool-projects/1610-remember-2/issues/3)

### CODE REVIEW

File: app/models/reminder.js [hunk 1]
```diff
@@ -4,6 +4,5 @@ export default DS.Model.extend({
   title: DS.attr('string'),
   date: DS.attr('date'),
   notes: DS.attr('string'),
-  pinned:
-  DS.attr('boolean')
+  pinned: DS.attr('boolean', { defaultValue: false })
```
Comment (martensonbj) [inline app/models/reminder.js:7]: 👍

File: app/router.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders-list', { path: '/' })
+  this.route('reminders-list', { path: '/reminders' })
```
Comment (martensonbj) [inline app/router.js]: The route here should simply be 'reminders', the path should be '/'. This indicates that when the path matches '/',  ember will look for the `reminders` route which will hand it the model it needs to render.

**Review by martensonbj — CHANGES_REQUESTED** [review]
Moooorrrre changes please

**Comment by dylanavery720** [comment]
@martensonbj @Tman22  Hey! We made the requested change. Let us know if you need anything else before you merge. Thank you.  


 🍌    :gorilla:

**Comment by martensonbj** [comment]
:shipit:

**Pull request merged**